plugins/out_http/http.h

```diff
@@ -29,6 +29,7 @@
 
 #define FLB_JSON_DATE_DOUBLE      0
 #define FLB_JSON_DATE_ISO8601     1
+#define FLB_JSON_DATE_EPOCH_INT   2
```

Comment:
It's not super clear what these are for our how they're used, so hopefully I got this right.